### DISCUSSION TITLE
how do I read requests into the network?

### DISCUSSION THREAD

**Discussion by FreeM1ne** [kaliiiiiiiiii/Selenium-Driverless#193] [Q&A] [answered]
how do I read requests into the network?

**Comment by ttlns** [comment]
Not sure what you're refering to? Can you provide an example (pseudo) script on what you'd like to do?

**Comment by FreeM1ne** [comment]
How do I get these requests through the code? I don't want to change them, just get their values
![image](https://github.com/kaliiiiiiiiii/Selenium-Driverless/assets/83845767/c7cbf1fa-33a5-4ac3-87ff-4cf20dca7a08)

**Reply by ttlns** [reply, marked as answer]
see requests interception in the readme. Next time, maybe read through first:)

**Comment by ttlns** [comment]
@FreeM1ne see https://kaliiiiiiiiii.github.io/Selenium-Driverless/api/RequestInterception/
